src/natcap/invest/validation.py

```diff
@@ -833,8 +833,19 @@ def _wrapped_validate_func(args, limit_to=None):
             assert isinstance(key, str), (
                 'All args keys must be strings.')
 
+        # Pytest in importlib mode makes it impossible for test modules to 
+        # import one another. This causes a problem in test_validation.py,
+        # which gets imported into itself here and fails.
+        # Since this decorator might not be needed in the future,
+        # just ignore failed imports; assume they have no ARGS_SPEC.
+        try:
+            model_module = importlib.import_module(validate_func.__module__)
+        except:
+            LOGGER.warning('Unable to import module %s: assuming no ARGS_SPEC.'
+                            % validate_func.__module__)
```

Comment:
Ah, sorry, this is a pretty trivial request, but I suspect a linter will suggest that we pass `validate_func.__module__` as a parameter rather than through string formatting.  I used to think that the parameter-passing approach here was the recommended way, but now I'm not sure where I saw that and suspect it came from pylint or the like.  In any case, the parameter-passing approach is how most non-`f`-string log messages are handled throughout InVEST, so I think this could be worth changing for the sake of consistency.

And as a side note, I had previously thought that the thing I'm suggesting was (for security purposes) the recommended approach for logging, but that doesn't seem to be the case.  All that `logging` does behind the scenes (by default, anyways) is [exactly what you have here, just a `%`-based parameter substitution](https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L363).  The one hypothetical advantage to passing args by parameter is that it does give us the _option_ down the road to sanitize the parameters somehow (like if we're dealing with user input in a webapp) before sending it to a remote logging server or something.

Reply:
Interesting about what `logging` is doing! I put it in a parameter instead of string formatting